topics/transcriptomics/tutorials/de-novo/tours/tour.yaml

```diff
@@ -77,21 +77,21 @@ steps:
       href="https://zenodo.org/record/583140">Zenodo</a>.
     placement: right
     textinsert: >-
-      https://zenodo.org/record/583140/files/G1E_rep1forward_read%28SRR549355_1%29
+      https://zenodo.org/record/583140/files/G1E_rep1_forward_read%28SRR549355_1%29
 
-      https://zenodo.org/record/583140/files/G1E_rep1reverse_read%28SRR549355_2%29
+      https://zenodo.org/record/583140/files/G1E_rep1_reverse_read%28SRR549355_2%29
 
```

Comment:
This link for for me: https://zenodo.org/record/583140/files/G1E_rep1_reverse_read_%28SRR549355_2%29


It seems there is a second _ missing after `read`

Reply:
For the rest as well ...